soc/arm/atmel_sam0/common/adc_fixup_sam0.h

```diff
@@ -141,4 +141,19 @@
 #  endif
 #endif
 
+/*
+ * The following MCLK clock configuration fix-up symbols map to the applicable
+ * APB-specific symbols, in order to accommodate different SoC series with the
+ * ADC core connected to different APBs.
+ */
+#ifdef MCLK_APBDMASK_ADC
+#  define MCLK_ADC (&MCLK->APBDMASK.reg)
+#endif
+#ifdef MCLK_APBDMASK_ADC0
+#  define MCLK_ADC (&MCLK->APBDMASK.reg)
+#endif
+#ifdef MCLK_APBCMASK_ADC0
+#  define MCLK_ADC (&MCLK->APBCMASK.reg)
```

Comment:
Please adjust all three to remove the ref-/de-reference - copy `ADC_SYNC()` (above, line 11), for consistency.

Reply:
@attie-argentum 
I move defines to 
```
#if defined(MCLK)
```
and there adjusted the same as above defines. Please check it.